spec/factories/users.rb

```diff
@@ -1,5 +1,7 @@
 FactoryGirl.define do
   factory :user do
+    first_name { Faker::Name.first_name }
+    last_name { Faker::Name.last_name }
     email { Faker::Internet.email }
```

Comment:
The email field should be built from first_name and include factory_girl sequence to avoid duplicates.